README.md

```diff
@@ -1 +1,83 @@
 # microcosm-metrics
+
+Opinionated metrics configuration.
+
+Designed primary to support [DataDog statsd](http://docs.datadoghq.com/guides/dogstatsd/), but some
+effort has been made to support [vanilla statsd](https://github.com/etsy/statsd).
+
+
+## Usage
+
+ 1. Configure a metrics client:
+
+        # either use vanilla statsd
+        graph.use("statsd")
+
+        # or use the datadog implementation
+        graph.use("datadog_statsd")
+
+ 2. In either case, use the resulting `graph.metrics` client:
+
+        metrics.increment("foo")
+
+
+## Decorators
```

Comment:
This section is new, as is all of the `metrics_counting` and `Classifier` support.